docs/2-techdoc/62304/checklist-software-release.md

```diff
@@ -24,22 +34,22 @@ The following documents are up to date:
-| ---- | --- | --- | ------- |
-| Device Description |  |  |  |
-| Clinical Evaluation |  |  |  |
-| Declaration of Conformity |  |  |  |
-| (...) |  |  |  |
+| Item                      | Yes | No  | Comment |
+| ------------------------- | --- | --- | ------- |
+| Device Description        |     |     |         |
+| Clinical Evaluation       |     |     |         |
+| Declaration of Conformity |     |     |         |
+| (...)                     |     |     |         |
 
 The following activities have been performed:
 
-| Item | Yes | No | Comment |
-| ---- | --- | --- | ------- |
-| All relevant functionalities of the software have been specified; the Software Requirement List is complete and has been reviewed. |  |  |  |
-| All relevant risks (including risks of known anomalies) have been evaluated; the Risk Management Report is complete. |  |  |  |
-| Verification (as software system testing) has been completed. |  |  |  |
-| Design control traceability is ensured:<br>Stakeholder requirements can be traced to software requirements.<br>Software requirements can be traced to system tests.<br>Software requirements can be traced to software code implementation / software code reviews.<br>Software requirements can be traced to risks and risk control measures.<br>Stakeholder requirements can be traced to usability tests.<br>Hazard-related use scenarios can be traced to usability tests.<br>Hazard-related use scenarios can be traced to risks and risk control measures. |  |  |  |
-| A version number as defined in the Software Development Plan has been assigned and added as a tag to git. |  |  |  |
-| Software is registered with a Notified Body. |  |  |  |
-| If release includes substantial change: Notified Body has been informed. |  |  |  |
-| Label is applied correctly including CE marking. |  |  |  |
+| Item                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | Yes | No  | Comment |
+| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --- | --- | ------- |
```

Comment:
this looks weird, and doesn't line up with the Item/yes/no/comment particularly well, even allowing for the white space showing after Item.  If it renders correctly, then no issue

Reply:
I suspect these are due to word wrap on your screen. If word wrap is turned off they should all line up after the auto-formatter has run...

Reply:
When viewing locally, with word-wrap turned off it looks like the following:

![image](https://user-images.githubusercontent.com/6559099/191214821-59584c80-e1d5-4ab2-ad59-c80ce658b407.png)

![image](https://user-images.githubusercontent.com/6559099/191214896-52cbc721-0b67-446d-971f-43c555334ed3.png)

I'll be pulling tables out into csv files when I step through the templates (which will fix this up)